### ISSUE TITLE
Web page, example gallery

### ISSUE THREAD

**Issue by bmcfee** [librosa/librosa#301]
What do people think about building up a simple home page for librosa?  I'm thinking something in the vein of [d3js.org](http://d3js.org/) but much simpler, and providing a handful of embedded (crowd-sourced) notebooks demonstrating various functionality.

This probably depends on #240.

**Comment by bmcfee** [comment]
Further thoughts:

- Gallery examples should be self-contained notebooks, possibly sourcing content off the web.
- It would be nice to have examples that reproduce certain published methods, e.g.:
  - superflux onsets (boeck)
  - harmonic-percussive-residual source separation (dreidger)
  - repet-sim (rafii & pardo) + weighted enhancement (fitzgerald)
  - non-local means for chroma enhancement (cho)
  - scale-transform rhythm analysis (holzapfel, prockup)
  - laplacian segmentation (mcfee)
  - etc etc etc....

**Comment by dpwe** [comment]
FWIW, e4896 now has several librosa-based practicals:
https://www.ee.columbia.edu/~dpwe/e4896/practicals.html#prac09 onward.

  DAn.

**Comment by bmcfee** [comment]
Awesome, thanks!

**Comment by bmcfee** [comment]
Repos are now in place for this ; migrating the issue accordingly.
